src/docs/cookbook/animation/opacity-animation.md

```diff
@@ -209,7 +218,7 @@ class _MyHomePageState extends State<MyHomePage> {
         },
         tooltip: 'Toggle Opacity',
         child: Icon(Icons.flip),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
```

Comment:
I removed this comment because it made it easier for the code excerpt script to extract the code sample